daemon/health.go

```diff
@@ -156,6 +157,7 @@ func monitor(d *Daemon, c *container.Container, stop chan struct{}, probe probe)
 			ctx, cancelProbe := context.WithTimeout(context.Background(), probeTimeout)
 			results := make(chan *types.HealthcheckResult)
 			go func() {
+				healthCheckCounter.Inc()
```

Comment:
The convention is to have a total and a failed counter, it's easier to work with.